examples/services.ml

```diff
@@ -1,7 +1,7 @@
 open Lwt
-open V1_LWT
+open Mirage_types_lwt
```

Comment:
I think that would be better, but this can be done that progressively later on (e.g. it is not a blocker for me for the release). Obviously it is better to have something working first :-)